app/src/main/java/com/github/libretube/ui/fragments/ChannelFragment.kt

```diff
@@ -29,7 +30,7 @@ class ChannelFragment : BaseFragment() {
 
     private var channelId: String? = null
     private var channelName: String? = null
-
+    private lateinit var shareData: ShareData
```

Comment:
That variable doesn't need to be initialized here, it should be sufficient to just initialize where it's being used below